raycicmd/make.go

```diff
@@ -61,6 +62,20 @@ func parsePipelineFile(file string) (*pipelineGroup, error) {
 	return g, nil
 }
 
+func getTagFilter(config *config) ([]string, error) {
+	tagFilterScript := config.TagFilterScript
+	if tagFilterScript == "" {
+		return nil, nil
+	}
+	cmd := exec.Command("bash", "-c", tagFilterScript)
```

Comment:
could you add a TODO here to put the execution in an unprivileged sandbox or something in the future?

also use `-elic` as the flag?